sapphire/simulations/gammas.py

```diff
@@ -60,30 +61,30 @@ def compton_energy_transfer(gamma_energy):
     return compton_edge(gamma_energy) * conversion_factor
 
 
-def dsigma_dT(E, T):
```

Comment:
I had the same problem with the changes to the landau arguments. I changed `Delta` to `delta`, which no longer matches the symbol used in Fokkema2012 eq 2.9 and 2.10.

In these cases I would prefer a 'phrase' which clarifies what the variable stands for. I am open to suggestions, you know more about this specific equation.